examples/notebooks/armory_colab_example.ipynb

```diff
@@ -0,0 +1,1232 @@
+        "from charmory.track import track_init_params, track_params\n",
+        "from charmory.utils import Unnormalize\n",
+        "import mlflow\n",
+        "from PIL import  Image\n",
+        "import os"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tMbJc0qCzOAu"
+      },
+      "source": [
+        "### We need to bring in the model and attach wrappers to run in this example"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "YSrkcW1Gbvvt",
+        "outputId": "44226ee0-1905-4bff-a215-42fa12629212"
+      },
+      "outputs": [],
+      "source": [
+        "model = JaticImageClassificationModel(\n",
+        "    track_params(AutoModelForImageClassification.from_pretrained)(\n",
+        "        \"farleyknight-org-username/vit-base-mnist\"\n",
+        "    ),\n",
+        ")\n",
+        "classifier = track_init_params(PyTorchClassifier)(\n",
+        "    model,\n",
+        "    loss=torch.nn.CrossEntropyLoss(),\n",
+        "    optimizer=torch.optim.Adam(model.parameters(), lr=0.003),\n",
+        "    input_shape=(3, 224, 224),\n",
+        "    channels_first=True,\n",
+        "    nb_classes=10,\n",
+        "    clip_values=(-1, 1),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 88
+        },
+        "id": "hIOG76MJAPVA",
+        "outputId": "c0d75be3-0e0c-4f0f-c354-ff6c03165800"
+      },
+      "source": [
+        "\n",
+        "For this section of code we create two variables model and classifier.\n",
+        "- The model variable uses AutoModelForImageClassification.from_pretrained which takes in a huggingface model card name as a variable. This retrieves the model from\n",
+        "  huggingface that we will use for this example. The track_params is a wrapper that stores the values of the model to be displayed and tracked by mlflow. Lastly,\n",
+        "  the JaticImageClassificationModel is another wrapper to make the model a jatic image classification class. This allows the model to have a standard output like other\n",
+        "  jatic image classification models.\n",
+        "- The classifier function wraps the model in a class to be useable by the ART library. It is a wrapper to make the model in a pytorch framework. It takes in as\n",
+        "  arguments the model, loss function, optimizer, input image sizes, bool value for RGB value being first, number of classes model predicts on, and clip value which is\n",
+        "  the values that will be the min and max values of the input after scaling. The track_params is another wrapper that stores the values of the model to be displayed\n",
+        "  and tracked by mlflow.\n",
```

Comment:
```suggestion
        "  the values that will be the min and max values of the input after scaling. We use `track_init_params` so that the constructor parameters for the ART wrapper are also tracked in MLflow.",
```